src/Pokemon.js

```diff
@@ -1,30 +1,112 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import './App.css';
+import Rating from '@mui/material/Rating';
+import { useParams } from 'react-router-dom';
+import { getPokemon } from './Frontend';
+import { getPokemonReviews, getUserReview, addReview } from './Backend';
+
+function Pokemon({ userdata }) {
+  const { id } = useParams();
+  const [RatingValue, setRatingValue] = useState(0);
+  const [PokemonInfo, setPokemonInfo] = useState({});
+  const [TotalReview, setTotalReview] = useState([]);
+  const [UserReview, setUserReview] = useState([]);
+  const ReviewTitle = useRef();
+  const ReviewBody = useRef();
+
+  useEffect(() => {
+    getPokemon(id).then((data) => {
+      setPokemonInfo({
+        name: data['name'],
+        pic: data['sprites']['other']['dream_world']['front_default'],
+      });
+    });
+  }, []);
+
+  useEffect(() => {
+    getPokemonReviews(id).then((data) => {
+      console.log(data)
+      if (data === null) {
+        setTotalReview(data);
+      } else {
+        setTotalReview(data["reviews"]);
+      }
+    });
+
+    getUserReview(userdata['username'], id).then((data) => {
+      console.log(data);
+      if (data === null) {
+        setUserReview(data);
+      } else {
+        setUserReview(data["reviews"][0]);
+      }
+    });
+  }, [setUserReview]
+  );
+  console.log(TotalReview);
+  console.log(UserReview);
```

Comment:
Before sunday midnight we should hunt down all of our console.log statements and print statements and comment them out

Reply:
Sure!